docs/sphinx/tiling.rst

```diff
@@ -0,0 +1,105 @@
+(131072×131072).  Different pixel types have different storage sizes,
+with consequent changes in storage size and upon performance.  These
+are shown on the panels as separate lines.
+
+These plots were computed using :download:`an R script
+<plots/tiling.R>`, based upon empirical testing with a `TIFF tiling
+benchmark
+<https://github.com/openmicroscopy/ome-files-performance/commit/8f6a8f3b25fd3761967b74366102466a590009f7>`__.
+
+.. figure:: plots/tile-file-size.svg
+   :scale: 100 %
+   :alt: Tile file size
+
+   The size of the TIFF file varies with different tile sizes and
+   pixel types.  Tile sizes which overlap the image border use more
+   space, which is most apparent with smaller images where it
+   comprises a greater proportion of the total file size.  Lower tile
+   sizes show a small increase in file size due to the storage of the
+   tile offsets and sizes.
+
+.. figure:: plots/tile-count.svg
+   :scale: 100 %
+   :alt: Tile count
+
+   As the tile size increases, the total number of tiles decreases.
+   The wastage due to image overlap becomes less apparent as the image
+   size increases.
+
+.. figure:: plots/tile-write-time.svg
+   :scale: 100 %
+   :alt: Tile write time
+
+   This plot combines the data from the previous plots with the
+   timings derived from `this benchmark result
+   <https://github.com/openmicroscopy/ome-files-performance/raw/8f6a8f3b25fd3761967b74366102466a590009f7/analysis/tile-test-write-performance.pdf>`__.
+   It is not necessarily representative of the timings which would be
+   observed in reality, where write caching, buffer copies and other
+   confounding effects which are not modelled here may cause
+   additional variation.
+
+.. figure:: plots/strip-file-size.svg
+   :scale: 100 %
+   :alt: Strip file size
+
+   The size of the TIFF file varies with different strip sizes and
+   pixel types.  Strip sizes which overlap the image border use more
+   space, which is most apparent with smaller images where it
+   comprises a greater proportion of the total file size.  Lower strip
+   sizes show a small increase in file size due to the storage of the
+   tile offsets and sizes.
+
+.. figure:: plots/strip-count.svg
+   :scale: 100 %
+   :alt: Strip count
+
+   As the strip size increases, the total number of strips decreases.
+   The wastage due to image overlap becomes less apparent as the image
+   size increases.
+
+.. figure:: plots/strip-write-time.svg
+   :scale: 100 %
+   :alt: Strip write time
+
+   This plot combines the data from the previous plots with the
+   timings derived from `this benchmark result
+   <https://github.com/openmicroscopy/ome-files-performance/blob/8f6a8f3b25fd3761967b74366102466a590009f7/analysis/strip-test-write-performance.pdf>`__.
+   It is not necessarily representative of the timings which would be
+   observed in reality, where write caching, buffer copies and other
+   confounding effects which are not modelled here may cause
```

Comment:
*Not* strictly but it's certainly less common, enough that I noticed instead of focusing on the content. E.g., if I try googling `"modelled" site:gov` I get around 217k results, but for "modeled" I get 1,850k results. https://en.wiktionary.org/wiki/modeled says,

> Alternative forms
>     modelled (British)